modules/apache-airflow/templates/values.yaml

```diff
@@ -68,7 +68,7 @@ airflowHome: /opt/airflow
 defaultAirflowRepository: ghcr.io/sage-bionetworks-workflows/orca-recipes
 
 # Default airflow tag to deploy
-defaultAirflowTag: "0.2.1"
+defaultAirflowTag: "latest"
```

Comment:
I don't think that we should use "latest" here. I would rather that we be very intentional about what could be and what is deployed to the K8s cluster. I do not want us to run into a situation where "latest" is updated and portions of the Airflow cluster don't get updated. This can happen if pods get moved around to new nodes, where the image pull process would get the new "latest" image and be more up to date than other pods in the cluster.

Instead, we should use a specific version tag to ensure consistent deployments across the entire cluster.

Reply:
Makes sense. Just made that update.